Sources/Starknet/Data/Responses.swift

```diff
@@ -24,6 +24,15 @@ public struct StarknetFeeEstimate: Decodable, Equatable {
         case overallFee = "overall_fee"
         case feeUnit = "unit"
     }
+
+    public init(gasConsumed: Felt, gasPrice: Felt, dataGasConsumed: Felt, dataGasPrice: Felt, overallFee: Felt, feeUnit: StarknetPriceUnit) {
+        self.gasConsumed = gasConsumed
+        self.gasPrice = gasPrice
+        self.dataGasConsumed = dataGasConsumed
+        self.dataGasPrice = dataGasPrice
+        self.overallFee = overallFee
+        self.feeUnit = feeUnit
+    }
```

Comment:
Nit: 

Not sure this is absolutely necessary, but maybe it would be reasonable to give option for `overallFee` to be automatically calculated based on other arguments?

Reply:
Sure, good idea.